sdk/logical/request.go

```diff
@@ -220,6 +220,10 @@ type Request struct {
 	// this will be the sha256(sorted policies + namespace) associated with the
 	// client token.
 	ClientID string `json:"client_id" structs:"client_id" mapstructure:"client_id" sentinel:""`
+
+	// RequestSSCToken is the token that arrives on an inbound request, supplied
+	// by the vault user.
+	RequestSSCToken string
```

Comment:
Changing to `InboundSSCToken` ?

Reply:
Hm, actually since we're not putting the created SSCToken in the request anymore, I suppose we could do away with `Inbound`, or `Request`, etc., entirely. Though, I still think that since we do persist state in the request that specifying this to be the token that comes in with the initial request is valuable.